priv/apps/itest/lib/in_flight_exit_client.ex

```diff
@@ -341,4 +360,19 @@ defmodule Itest.InFlightExitClient do
     |> ABI.TypeDecoder.decode([:bool])
     |> hd()
   end
+
+  defp get_next_exit_from_queue() do
+    data = ABI.encode("getNextExit(uint256,address)", [Itest.Account.vault_id(Currency.ether()), Currency.ether()])
+    {:ok, result} = Ethereumex.HttpClient.eth_call(%{to: Itest.Account.plasma_framework(), data: Encoding.to_hex(data)})
+
+    case Encoding.to_binary(result) do
+      "" ->
+        _ = Logger.info("Empty exit queue while #{__MODULE__}.")
+
+      result ->
+        next_exit_id = hd(ABI.TypeDecoder.decode(result, [{:uint, 256}]))
+        exit_id = next_exit_id &&& (1 <<< 160) - 1
```

Comment:
yeah, good point! you reminded me I got this from slack or GH.
But hopefully this can get added into int. docs now: https://github.com/omisego/plasma-contracts/pull/557/files#r365761956